includes/wikia/tests/UserTest.php

```diff
@@ -178,4 +178,25 @@ public function testGetOptionShouldReturnPreferenceDataFromService() {
 		$this->assertEquals( "someLocalWikia1Value", $options[ "someLocalWikia1Pref" ] );
 		$this->assertArrayNotHasKey( "someLocalWikia2Pref", $options );
 	}
+
+	public function testGetUsernameShouldReturnAnonNameForUserIdZero() {
+		$this->assertEquals( 'anonName', User::getUsername( 0, 'anonName' ) );
+	}
+
+	public function testGetUsernameShouldReturnProvidedNameIfLookupIsDisabled() {
+		$this->mockGlobalVariable( 'wgEnableUsernameLookup', false );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "wgEnableUsernameLookup" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)